### DISCUSSION TITLE
How to limit amount of pages or only load a certain set of them?

### DISCUSSION THREAD

**Discussion by twan-2022** [area17/twill#1939] [Q&A]
I am using the pages module and my CMS contains a lot of pages, a few thousand. There is no pagination so it is very slow. I only need to edit a few of them sometimes so I would like to only load those pages. 

Is there an easy way to do this? Maybe overwrite a function in a controller to only load pages with certain id's?

**Comment by kallefrombosnia** [comment]
Pagination should be available on the module index view.
Docs:
```php
// Optional, specify number of items per page in the listing view (-1 to disable pagination)
// If you are implementing Sortable, this parameter is ignored given reordering is not implemented
// along with pagination.
protected $perPage = 20;
```

Do you maybe implement `Sortable` on your Model?

**Reply by twan-2022** [reply]
Yeah I had sortable in it but I didn't really need it for these pages so I removed it from the model and added the pagination. Thanks!
